Docs revamp! (#705)
grafana/tempo: docs/tempo/website/deployment/_index.md

# Deploying Tempo

Tempo can be easily deployed through a number of tools as explained in this document.

> > Note: The Tanka and Helm examples are equivalent.

Reviewer: ```suggestion
> **Note**: The Tanka and Helm examples are equivalent.
```

> > They are both provided for people who prefer different configuration mechanisms.

Reviewer 2: `prefer different` -> `prefer a different`

Author: `prefer different configuration mechanisms` is correct here

## Tanka / Jsonnet

> The jsonnet files required to deploy Tempo with Tanka are available here:

Reviewer: ```suggestion
The Jsonnet files that you need to deploy Tempo with Tanka are available here:
```

- [single binary](https://github.com/grafana/tempo/tree/main/operations/jsonnet/single-binary)
- [microservices](https://github.com/grafana/tempo/tree/main/operations/jsonnet/microservices)

> A few [examples](https://github.com/grafana/tempo/tree/main/example/tk) have been provided that use official jsonnet files.

Reviewer: ```suggestion
Here are a few [examples](https://github.com/grafana/tempo/tree/main/example/tk) that use official Jsonnet files.
```

> These show off the full range of configuration available to Tempo.

Reviewer: ```suggestion
Ther display the full range of configuration available to Tempo.
```
